### PR TITLE
Feature: RTT command verbose mode

### PR DESCRIPTION
**Pull request by boris-ilijic**
RTT command examines the final result with few iterations (default is 5).
It should be handy to be able to see RTT for each iteration, and for that purpose `--trace` switch is reused. This PR adds that logic.

Example:
```
nats rtt --trace 8  
                                                                                                                                             
RTT iterations for server: nats://127.0.0.1:4222
#1:	rtt=1.514372ms
#2:	rtt=1.924859ms
#3:	rtt=1.322852ms
#4:	rtt=1.31299ms
#5:	rtt=1.120805ms
#6:	rtt=1.121725ms
#7:	rtt=1.133196ms
#8:	rtt=1.321921ms

nats://127.0.0.1:4222:

   127.0.0.1: 1.34659ms
```

### CODE REVIEW

File: cli/rtt_command.go [hunk 1]
```diff
@@ -143,13 +143,22 @@ func (c *rttCmd) calcRTT(server string, copts []nats.Option) (string, time.Durat
...
+	}
 	for i := 1; i <= c.iterations; i++ {
 		rtt, err := nc.RTT()
 		if err != nil {
 			return "", 0, fmt.Errorf("rtt failed: %v", err)
 		}
 
 		totalTime += rtt
+		if opts.Trace {
+			fmt.Printf("#%d:\trtt=%s\n", i, rtt)
```
Comment (ripienaar) [inline cli/rtt_command.go:157]: can maybe pass `rtt` through `humanizeDuration()`

**Comment by boris-ilijic** [comment]
@ripienaar Applied HumanizeDuration formatting and signed commits also.

But output was better before if you asked me, now it has just ms precision:
```
RTT iterations for server: nats://127.0.0.1:4222
#1:	rtt=1ms
#2:	rtt=1ms
#3:	rtt=1ms
#4:	rtt=1ms
#5:	rtt=1ms
#6:	rtt=1ms
#7:	rtt=2ms
#8:	rtt=2ms

nats://127.0.0.1:4222:

   127.0.0.1: 1.405241ms
```
I'll remove it if you feel the same?

**Comment by ripienaar** [comment]
yeah I think you're right, lets remove again, sorry about that.

**Comment by ripienaar** [comment]
Thank you

**Pull request merged**